pkg/operator/ceph/operator.go

```diff
@@ -219,8 +219,7 @@ func (o *Operator) updateDrivers() error {
 	}
 
 	if !csi.CSIEnabled() {
-		logger.Infof("CSI driver is not enabled")
-		return nil
+		logger.Infof("CSI driver is disabled")
```

Comment:
need more logging about which driver is disabled and also `Infof` is not required

Reply:
I think we can remove this block from here because if CSI driver is disabled anyway we are logging that in `stopDrivers()` with better logging like which driver is disabled.

Reply:
I think we can remove `if !csi.CSIEnabled()` and, we can be logging if CSI driver is disabled in `stopDrivers()` ?

Reply:
looks good. we can remove it